src/coreclr/gc/gc.cpp

```diff
@@ -34777,7 +34777,7 @@ BOOL gc_heap::commit_mark_array_bgc_init()
             {
                 // For ro segments they could always be only partially in range so we'd
                 // be calling this at the beginning of every BGC. We are not making this
-                // more efficient right now - ro segments are currently only used by redhawk.
+                // more efficient right now - ro segments are currently only used by NativeAOT.
```

Comment:
@dotnet/gc this comment is obviously not true in 2022. How should this comment be written?